db/migrate/20200330153007_rename_courses_accrediting_provider_id_part1.rb

```diff
@@ -1,5 +1,9 @@
 class RenameCoursesAccreditingProviderIdPart1 < ActiveRecord::Migration[6.0]
   def up
+    unless column_exists?(:courses, :accredited_provider_id)
+      add_column :courses, :accredited_provider_id, :integer, null: true
+    end
```

Comment:
Should we add the reverse of this in the `down`?

Reply:
I don't think so because this migration wasn't supposed to add it and doing so could mess up the other migration getting rolled back (maybe). If it needs to be dropped it would be dropped by the `add_courses_accredited_provider_id` migration whichever order they are run in.